test/export-integration.test.js

```diff
@@ -0,0 +1,108 @@
+/*
+This file contains integration tests for the exportDataset function and are based on fixtures
+  in the fixtures directory. Each fixture contains a set of test cases that are run against the
+  exportDataset function and a mocked backend API with disabled network requests.
+*/
+
+const fs = require('fs')
+const yaml = require('yaml')
+const nock = require('nock')
+const path = require('path')
+const {afterAll, describe, expect, test} = require('@jest/globals')
+const exportDataset = require('../src/export')
+const rimraf = require('../src/util/rimraf')
+const sanity = require('@sanity/client')
+const {untarExportedFile, ndjsonToArray} = require('./helpers')
+const {newTestRunId, withTmpDir} = require('./helpers/suite')
```

Comment:
Could we sort these imports, or in another PR add support for prettier/eslint sorting them? :p